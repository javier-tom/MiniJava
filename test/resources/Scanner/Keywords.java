class Keywords {
    public static void main(String[] args) {

    }
}