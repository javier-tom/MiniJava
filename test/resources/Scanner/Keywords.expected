CLASS ID(Keywords) LCURLY PUBLIC STATIC VOID MAIN LPAREN STRING_TYPE LSQUARE RSQUARE ID(args) RPAREN LCURLY RCURLY RCURLY